docs/dependency-injection-liftweb-scala.adoc

```diff
@@ -0,0 +1,382 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+= Dependency Injection in Lift
+
+== Introduction
+
+Dependency injection is an important topic in the JVM world. With Java, we usually go with Guice
+or Spring whenever we need it. However, scala has certain advantages over Java, like functions,
```

Comment:
Scala should be capitalized. I would suggest rephrasing here to say, ‘Scala provides some unique advantages that allow many of the features needed for DI without requiring an entire DI framework’.